a3fe/run/system_prep.py

```diff
@@ -578,21 +607,29 @@ def heat_and_preequil_input(
     print(
         f"NPT equilibration for {cfg.runtime_npt} ps while restraining non-solvent heavy atoms"
     )
-    protocol = _BSS.Protocol.Equilibration(
-        runtime=cfg.runtime_npt * _BSS.Units.Time.picosecond,
-        pressure=1 * _BSS.Units.Pressure.atm,
-        temperature=cfg.end_temp * _BSS.Units.Temperature.kelvin,
-        restraint="heavy",
+    if not cfg.membrane_protein:
```

Comment:
Could maybe remove a bit of duplication here, e.g.
```
protocol_args = {runtime=cfg.runtime_npt * _BSS.Units.Time.picosecond ... etc}
if cfg.membrane_protein:
    protocol_args["restraint"] = "heavy"
protocol = _BSS.Protocol.Equilibration(**protocol_args)
```
Trying to make it clearer and easier to see what changes when you set `membrane_protein`, but maybe I'm just complicating!